lib/platform/gitlab/index.ts

```diff
@@ -162,6 +163,49 @@ export async function getJsonFile(
   return JSON.parse(raw);
 }
 
+function getRepoUrl(
+  repository: string,
+  res: HttpResponse<RepoResponse>
+): string {
+  const { gitUrl } = getAdminConfig();
+
+  if (gitUrl === 'ssh') {
+    logger.debug(`${repository} ssh URL = ${res.body.ssh_url_to_repo}`);
```

Comment:
throw config error if `res.body.ssh_url_to_repo` is null or undefined

Reply:
Can I simply add this?
```js
export const CONFIG_VALIDATION = 'config-validation';
```

Or use one of the existing ones?
```js
// Config Error
export const CONFIG_VALIDATION = 'config-validation';
export const CONFIG_PRESETS_INVALID = 'config-presets-invalid';
export const CONFIG_SECRETS_EXPOSED = 'config-secrets-exposed';
export const CONFIG_SECRETS_INVALID = 'config-secrets-invalid';
```